doc/source/whatsnew/v1.2.0.rst

```diff
@@ -13,6 +13,21 @@ including other versions of pandas.
 Enhancements
 ~~~~~~~~~~~~
 
+.. _whatsnew_120.improve_timeseries_accessor_naming:
```

Comment:
sorry to be a pain but could you do something short here like the whatsnew in this PR:
 https://github.com/pandas-dev/pandas/pull/9605/files